doc/user/content/concepts/views.md

```diff
@@ -15,104 +15,119 @@ aliases:
 Views represent queries that are saved under a name for reference. Views provide
 a shorthand for the underlying query.
 
-Materialize offers the following types of views:
-
 Type |
 -----|------------
-[**Non-materialized views**](#non-materialized-views) | Results are **not** persisted in durable storage. Non-materialized views can be indexed to maintain and incrementally update results in memory.
+[**Views**](#views) | Results are **not** persisted in durable storage. These views can be indexed to maintain and incrementally update results in memory.
 [**Materialized views**](#materialized-views) | Results **are** persisted and incrementally updated in durable storage. Materialized views can be indexed to maintain the results in memory.
 
 All views in Materialize are built by reading data from
 [sources](/concepts/sources) and other views.
 
-## Non-materialized views
-
-A non-materialized view, also generally referred to as a "view", saves a query
-under a name to provide a shorthand for referencing the query. The query is
-**not** executed during non-materialized view creation, and a non-materialized
-view does **not** persist its results in durable storage.
+## Views
 
-**However**, you can [index](/concepts/indexes/) a non-materialized view to
-**maintain and incrementally update view results** in memory within a cluster.
-This enables queries within that [cluster](/concepts/clusters/) to use the index
-to access view results from memory.  See [Indexes and non-materialized
-views](#indexes-and-non-materialized-views) for more information.
+A view saves a query under a name to provide a shorthand for referencing the
+query. Views can be referenced across [clusters](/concepts/clusters/).
 
-See also:
+During view creation, the underlying query is not executed , and the view
+results are not persisted in durable storage.
 
-- [`CREATE VIEW`](/sql/create-view)
+```mzsql
+CREATE VIEW my_view_name AS
+  SELECT ... FROM ...  ;
+```
 
-### Indexes and non-materialized views
+<red>However</red>, in Materialize, you can [index](/concepts/indexes/) a view
+to **maintain and incrementally update** view results in memory within a
+cluster.
```

Comment:
```suggestion
<red>However</red>, in Materialize, you can create an [index](/concepts/indexes/)
on a view to **maintain and incrementally update** its results in memory within
a cluster.
```

Reply:
Could it be more eye-popping to have this as an annotation? Possibly a new annotation type, _e.g._ **💡 TIP**, which we could reuse across the docs to highlight important concepts or best practices.

Reply:
Okay, you read my mind: https://github.com/MaterializeInc/materialize/pull/28410/files#diff-07c43ff89beedeabdcaa61a4bd4f772b47cf9b86fe243f7f1e540cf166de9b99